cli/src/main.rs

```diff
@@ -469,6 +476,16 @@ async fn main() -> Result<()> {
                 }
             }
         }
+        Command::Abandon { cluster, backends } => {
+            let cluster = ClusterName::new(&cluster);
+            for backend in backends {
+                let backend = BackendId::new(backend);
+                let lost_state_message =
+                    BackendStateMessage::new(BackendState::Lost, cluster.clone(), backend);
+
+                nats.publish_jetstream(&lost_state_message).await?;
```

Comment:
Can we confirm that publishing this message will result in the platform updating the postgres DB with the new Lost state?

Reply:
Hmm I can confirm that it gets rid of locks, but I didn't check what it would do to the platform, I'll run it locally. iirc the platform listens to the drones for statuses right? in that case it wouldn't

Reply:
Actually, I think we might be good - looks like p2 listens to the `BackendStateMessage` subject - https://github.com/drifting-in-space/p2/blob/main/jamsocket/src/platform/mod.rs#L102C29-L102C29